wiki/Visual_Settings/en.md

```diff
@@ -1,9 +1,10 @@
 # Visual Settings
 
-![Example of Visual Settings](img/VisualSettings.jpg "Example of Visual Settings")
+![Example of Visual Settings](./VisualSettings.jpg "Example of Visual Settings")
 
--   Visual Settings is below the screen, a few seconds before the beatmap starts. Having it open delays the beatmap from starting until your mouse pointer leaves it.
-    -   You can pause the game to open Visual Settings (Does not work on [Multi](/wiki/Multi)).
+**Visual Settings** is a hidden setting located at the bottom of a beatmap's playfield a few seconds before the beatmap starts. When the `Visual Settings` is shown, the beatmap will be delayed until the osu!cursor left the `Visual Settings` screen.
+
+Alternatively, it is possible to pause to access `Visual Settings` but the pause method will not work in a [Multi](/wiki/Multi) match due to the keybind to pause is rebinded to quit the room instead when pressed twice.
```

Comment:
Alternatively, it is possible to access `Visual Settings` by pausing the game, but it will not work in a Multi match due to the pause keybind being rebinded to quitting the room instead.